screens/Request/RequestScreenController.ts

```diff
@@ -15,11 +16,14 @@ import {
 import { selectVidLabel } from '../../machines/settings';
 import { MainRouteProps } from '../../routes/main';
 import { GlobalContext } from '../../shared/GlobalContext';
+import BluetoothStateManager from 'react-native-bluetooth-state-manager';
 
 export function useRequestScreen({ navigation }: MainRouteProps) {
   const { appService } = useContext(GlobalContext);
   const requestService = appService.children.get('request');
   const settingsService = appService.children.get('settings');
+  const isActive = useSelector(appService, selectIsActive);
+  const isBluetoothDenied = useSelector(requestService, selectBluetoothDenied);
```

Comment:
Can you reference this variable in the returned object so we don't duplicate the selector? (in line 68)